### PR TITLE
Add AWS_SESSION_TOKEN and image ID info

### PR DESCRIPTION
**Pull request by paigehargrave**

### CODE REVIEW

File: modules/ROOT/pages/prereq-aws-terraform.adoc [hunk 1]
```diff
@@ -96,8 +97,12 @@ You can also include the following set of optional environment variables:
 docker load -i pcp-1.0.tar.gz
 ----
 
-. Record the image ID after running the `docker load` command. This is required in the following step.
-
+. Record the image ID after running the `docker load` command. Run the following command to retrieve the image ID, which is required for the next step:
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-aws-terraform.adoc]: No need to run gravity images here. The image ID you need to use in the next step shows up right after you run docker load in the previous step

**Review by alejandrorascovan — DISMISSED** [review]

File: modules/ROOT/pages/prereq-aws-terraform.adoc [hunk 2]
```diff
@@ -62,6 +62,7 @@ ssh -i 'anypoint.pem' ec2-user@18.221.155.87
 | AWS_KEY_NAME | Specifies the AWS SSH key. Do not include the `.pem` extension.
 | AWS_REGION | Specifies the AWS region where Terraform creates the cluster, for example, `us-east-2`. This value must have the same value as AWS_DEFAULT_REGION.
 | AWS_DEFAULT_REGION | Must be the same value as AWS_REGION.
+| AWS_SESSION_TOKEN | Specifies AWS temporary security credentials.
```
Comment (alejandrorascovan) [inline modules/ROOT/pages/prereq-aws-terraform.adoc]: Let's move up this line right after `AWS_SECRET_ACCESS_KEY` and for the description let's change it to:
`Specifies the AWS temporary session token if any`

Comment (paigehargrave) [reply modules/ROOT/pages/prereq-aws-terraform.adoc]: Got it - thank you!

File: modules/ROOT/pages/prereq-aws-terraform.adoc (hunk 1 above)
Comment (paigehargrave) [reply modules/ROOT/pages/prereq-aws-terraform.adoc]: Updated this - thank you!

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**